src/draganddrop.js

```diff
@@ -0,0 +1,21 @@
+export default class DragAndDrop {
+  constructor(idx) {
+    this.index = idx;
+  }
+
+  dragStart = () => {
+    document.querySelectorAll('.list-item')[this.index].addEventListener('dragstart', () => {
+      // console.log(document.querySelectorAll('.list-item')
+      // [this.index].firstElementChild.firstElementChild.nextElementSibling);
+      document.querySelectorAll('.list-item')[this.index].firstElementChild.firstElementChild.nextElementSibling.value = '';
+    });
+  }
+
+  dragOver = () => {
+
+  }
+
+  drop = () => {
+
+  }
+}
```

Comment:
- Nice idea trying to implement the drag and drop features. However, I think you should remove the code from your codebase as it is not being used.